crates/parameters/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+edition = "2021"
+homepage = "https://github.com/hulks/hulk"
+license = "GPL-3.0-only"
+name = "parameters"
+version = "0.1.0"
+
+[dependencies]
+color-eyre = {workspace = true}
+log = {workspace = true}
```

Comment:
Do you need `color_eyre` and `log`?

Reply:
Log is used in `directory.rs` for some debug messages.